dataflow/src/org/checkerframework/dataflow/analysis/Store.java

```diff
@@ -54,6 +54,25 @@
     S leastUpperBound(S other);
 
     /**
+     * Compute an upper bound of two stores that is wider than the least upper bound of the two
+     * stores.
```

Comment:
Any advice on how much wider? Is it good enough to just return TOP?
When is the method used?